<launch>
     <node name="duo3d" pkg="duo3d_driver" type="duo3d_driver_external_calibration" output="screen">
                <param name="frame_rate" value="10.0"/>
                <rosparam param="image_size">[752, 480]</rosparam>
                <param name="dense3d_license" value="XX"/>
		<param name="auto_exposure" value="true"/>
		<param name="vertical_flip" value="true"/>
		<param name="led" value="30"/>
    </node>
	<node pkg="rqt_gui" type="rqt_gui" name="rqt_gui" args="--perspective-file $(find duo3d_driver)/launch/rqt/stereo_view.perspective"/>
</launch>
